lib/src/json_schema/validator.dart

```diff
@@ -162,38 +187,40 @@ class Validator {
     final exclusiveMaximum = schema.exclusiveMaximum;
     final exclusiveMinimum = schema.exclusiveMinimum;
 
+    final warnOrErr = _errFuncForVocabulary(SupportedVocabularies.VALIDATION);
```

Comment:
@michaelcarter-wf I chatted with @robbielamb-wf  about what we should do if there ordinarily would have been a validation error, but it was skipped due to the vocabulary not being in use. We decided a warning was probably sane.

Reply:
Maybe we should talk through this, but I would think the opposite. If you explicitly don't depend on a vocab, would you really want to see warnings for it?

Reply:
Thinking through this - if you are running validation, you're interested in finding issues with your data. You may or may not have any control over the schema definition. If the schema author defined something like an `exclusiveMinimum`, but is using a metaschema that doesn't specify that vocabulary... the schema author is the one who probably has an issue, right? I guess the warning would let the data owner know "your data validates against the actual schema, but perhaps not the _intention_ of the schema." Is that worth getting a warning for?

Reply:
Chatted live - we'll drop the warnings for now for simplicity :man_shrugging: